### ISSUE TITLE
Accessibility: Unable to move VoiceOver cursor to DetailView

### ISSUE THREAD

**Issue by vincode-io** [brentsimmons/NetNewsWire#517]
When focus is in the Timeline it isn't possible to move the VoiceOver cursor to the detail view.

Recommended resolution: 
Enable the right arrow key as a keyboard shortcut that moves focus to the detail view.

**Comment by vincode-io** [comment]
Related: we should also enable the left arrow key in DetailView to get back to the Timeline.

**Comment by brentsimmons** [comment]
The tab key should work to move focus. (Right-arrow means open-in-browser.)

**Comment by vincode-io** [comment]
The tab key does shift focus to the detail view.  Not sure how I missed that.  

We still need a way to return focus back to the timeline from the detail view.  Since we already use the arrow keys to move between the sidebar and the timeline, I think this is the most intuitive way to shift the focus back and forth between the timeline and detail view.

Since command-right-arrow opens in the browser, I think that right-arrow fits well with moving focus to the detail web view.

**Comment by brentsimmons** [comment]
Sounds fair. Timeline -> Detail: right arrow. Detail -> Timeline: left arrow.

**Comment by vincode-io** [comment]
So I implemented this today and discovered something about VoiceOver.  When it is focused on web content, VoiceOver takes over the arrow keys.  This means that the leftarrow to return from the web content isn't possible to implement.  VoiceOver always handles the leftarrow event.

I also learned more about how VoiceOver is supposed to be used while doing this.  Using control-option with the arrow keys is how you are supposed to navigate through various views in a window.  So using contol-option-rightarrow and control-option-leftarrow will move you between the sidebar, timeline, and detail view already.

I no longer believe this Issue to be an issue and recommend closing it with no code changes.

**Comment by brentsimmons** [comment]
Sounds good. Thanks, Maurice!
